results/intfloat__multilingual-e5-small/ContractQALegalBenchPC.json

```diff
@@ -0,0 +1,49 @@
+{
+  "dataset_revision": "12ca3b695563788fead87a982ad1a068284413f4",
+  "mteb_dataset_name": "ContractQALegalBenchPC",
```

Comment:
This task looks a bit trivial, at least both models get more than 90% of accuracy, maybe we shouldn't keep it